### DISCUSSION TITLE
Translate labels on photoprism?

### DISCUSSION THREAD

**Discussion by juanpeich** [photoprism/photoprism#3576] [Q&A]
Is it possible to translate the tags generated by the Google tensor into other languages ​​(in my case Spanish)? Tags like "portrait", "landscape", etc.

 If it can't be done from configuration, could it be changed manually in a file that has these tags? It would be a matter of translating one by one. Thanks in advance JuamPe

**Comment by graciousgrey** [comment]
Labels and Locations can not be translated yet.  However we do have open feature requests for it:
- https://github.com/photoprism/photoprism/issues/485
- https://github.com/photoprism/photoprism/issues/883

As a workaround you can rename labels:
- https://docs.photoprism.app/user-guide/organize/labels/#rename-labels
